spec/javascripts/support/jasmine_helper.rb

```diff
@@ -0,0 +1,14 @@
+# Use this file to set/override Jasmine configuration options
+# You can remove it if you don't need it.
+# This file is loaded *after* jasmine.yml is interpreted.
+#
+# Example: using a different boot file.
+# Jasmine.configure do |config|
+#    config.boot_dir = '/absolute/path/to/boot_dir'
+#    config.boot_files = lambda { ['/absolute/path/to/boot_dir/file.js'] }
+# end
+#
+# Prevent PhantomJS auto install, uses PhantomJS already on your path.
+Jasmine.configure do |config|
+  config.prevent_phantom_js_auto_install = false
```

Comment:
good catch, thanks. Fixed it in https://github.com/fs/rails-base/commit/e31337a0f9ab8d36c51afeccc2b2b8b2876c5337